schemas/database/013_batches_and_transactions.sql

```diff
@@ -0,0 +1,25 @@
+-- Create ENUM for prover type
+CREATE TYPE batch_type_enum AS ENUM ('Insertion', 'Deletion');
```

Comment:
Postgres enums are tricky as you can't add a variant to them later down the line.

I would suggest changing it to a number or string.

Another option is a table with allowed values but that's a bit overkill imo.

Reply:
I have seen we were using it for "prover_type" (file 008_prover_type.sql). What we prefer? Do it like for prover or just use String field and parse on rust side?